### PR TITLE
ZWEB-3674 JF Make All Library Packages Fixed & Fix Security Issues

### PR DESCRIPTION
**Pull request by Jacob-Friesen**
See Internal PR: https://github.com/zimit-io/zimit-webapp/pull/3874

**Replication**
1. Run the install and audit: `npm install && npm audit`
1. Run the tests: `npm test`

### CODE REVIEW

File: lib/security.service.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const _ = require('lodash');
 
 const UserPolicy = require('./user-policy.model');
 const initSecurity = require('./security-definition.service');
-const zlog = require('zlog');
+const zlog = require('zimit-zlog');
```
Comment (etelmon) [inline lib/security.service.js:9]: You changed the log here but not here: spec/security.service.spec.js

File: package.json [hunk 2]
```diff
@@ -16,16 +16,16 @@
...
+        "promise": "7.1.1",
+        "request": "2.88.2",
+        "uuid": "2.0.1",
+        "zimit-zlog": "1.0.6"
     },
     "devDependencies": {
-        "jasmine": "^2.4.1",
-        "zerv-core": "git://github.com/z-open/zerv-core#1.0.3",
+        "jasmine": "3.5.0",
+        "zerv-core": "git://github.com/z-open/zerv-core#1.2.3",
```
Comment (etelmon) [inline package.json:28]: new zerv core and sync releases...

File: package.json [hunk 3]
```diff
@@ -16,16 +16,16 @@
     },
     "license": "MIT",
     "dependencies": {
-        "express": "^4.14.0",
-        "lodash": "^4.15.0",
-        "promise": "^7.1.1",
-        "request": "^2.74.0",
-        "uuid": "^2.0.1",
-        "zlog": "git://github.com/z-open/zlog#1.0.0"
+        "express": "4.17.1",
```
Comment (etelmon) [inline package.json:19]: Express is not used.

File: package.json [hunk 4]
```diff
@@ -16,16 +16,16 @@
...
+        "lodash": "4.17.15",
+        "promise": "7.1.1",
+        "request": "2.88.2",
```
Comment (etelmon) [inline package.json:22]: Request is not used

File: package.json [hunk 5]
```diff
@@ -16,16 +16,16 @@
     },
     "license": "MIT",
     "dependencies": {
```
Comment (etelmon) [inline package.json:18]: For consistency with other libs, pls add linting.

File: lib/security.service.js (hunk 1 above)
Comment (Jacob-Friesen) [reply lib/security.service.js:9]: fixed.

File: package.json (hunk 3 above)
Comment (Jacob-Friesen) [reply package.json:19]: ok. It was there before, so I kept it. If you are sure, then I will remove it.

File: package.json (hunk 4 above)
Comment (Jacob-Friesen) [reply package.json:22]: ok. It was there before, so I kept it. If you are sure, then I will remove it.

File: package.json (hunk 5 above)
Comment (Jacob-Friesen) [reply package.json:18]: Not if we don't have it already. This PR needs to remain focused only on security and package upgrades. Linting can be added in a future PR (which will not have to be added to the main application).

File: package.json (hunk 4 above)
Comment (etelmon) [reply package.json:22]: pls remove.

File: package.json (hunk 3 above)
Comment (etelmon) [reply package.json:19]: pls remove. I use npm-check and did run the test.

File: package.json (hunk 5 above)
Comment (etelmon) [reply package.json:18]: ok then.

File: lib/security.service.js (hunk 1 above)
Comment (etelmon) [reply lib/security.service.js:9]: ok

File: package.json (hunk 2 above)
Comment (Jacob-Friesen) [reply package.json:26]: The `zerv-core` one seems to be the latest: https://github.com/z-open/zerv-core/releases. I had to push a version change to `https://github.com/z-open/zerv-sync/commit/c855e641def17b804487ce5d596cdac6343dd330`. Then I included that one in this library.

File: package.json (hunk 4 above)
Comment (Jacob-Friesen) [reply package.json]: It was removed in the latest commit.

File: package.json (hunk 3 above)
Comment (Jacob-Friesen) [reply package.json]: It was removed in the latest commit.

File: package.json (hunk 2 above)
Comment (etelmon) [reply package.json:26]: I see. okay then.

**Pull request merged**